accounts/tests.py

```diff
@@ -332,6 +332,85 @@ def test_user_save(self):
         self.assertEqual(Profile.objects.filter(user=u).exists(), True)
         u.save()  # Check saving user again (with existing profile) does not fail
 
+    def test_user_registration(self):
+        RegistrationForm.captcha_key = ''  # Override this property so that captcha is not enabled in the tests
+        # NOTE: using override_settings and setting RECAPTCHA_PUBLIC_KEY to '' won't work because RegistrationForm is
```

Comment:
You could fix this by setting `RegistrationForm.captcha_key` in the form's `__init__`, or by reading from settings directly where you need to use it.
Ideally you would revert this setting back at the end of the test